bootstrap/terraform/values/prometheus.yaml

```diff
@@ -11,6 +11,13 @@ prometheus:
       podMetricsEndpoints:
         - port: "metrics"
       selector: {}
+    - name: "argocd"
+      namespaceSelector:
+        matchNames:
+          - "argocd"
+      podMetricsEndpoints:
+        - port: "metrics"
+      selector: {}
```

Comment:
Crossplane has a podmonitor because the family provider pod don't have a service. Since ArgoCD has a service, it is better to add a servicemonitor for it.

Reply:
fixed